include/deal.II/base/mpi.h

```diff
@@ -708,6 +708,44 @@ namespace Utilities
            const T &          object_to_send,
            const unsigned int root_process = 0);
 
+    /**
+     * Given a partitioned index set space, compute the owning MPI process rank
+     * of each element of a second index set according to the partitioned index
+     * set. A natural usage of this function is to compute for each ghosted
+     * degree of freedom the MPI rank of the process owning that index.
+     *
+     * The user is responsible to partition the input index set space correctly.
+     * Otherwise, there are no limitations: there is no need in partitioning
+     * the index set space continuously. Furthermore, there are no limitations
+     * on the second index set as long as the size matches the first one:
+     * in the case that the second index set also contains locally
+     * owned indices, these indices will be treated correctly and the rank of
+     * this process is returned for those entries.
```

Comment:
Does this second argument have to be the same on every process? If not, it might be worth stating this explicitly here.